manifests/b/byCrookie/TypeCode/1.1.11/byCrookie.TypeCode.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: byCrookie.TypeCode
+PackageVersion: 1.1.11
+InstallerType: msi
+InstallModes:
+  - silent
```

```suggestion
  - silent
AppsAndFeaturesEntries:
  - DisplayVersion: 1.1.11.0
```

I see that you market the version as `1.1.11` in your GitHub Releases, but WinGet requires the version in Registry / Control Panel / Apps&Features for matching upgrades. One resolution is to use the "actual" DisplayVersion field of Registry in place of the PackageVersion field in the manifest.
Another resolution is to use the "marketing" version in the PackageVersion field (this version will be shown to the user for winget install), but use `AppsAndFeaturesEntries` and explicitly provide the "actual" DisplayVersion for WinGet to match upgrades. I've suggested the latter here.